### PR TITLE
Refer to R instead of Python in the introduction.

### LINKED ISSUE #3: The introduction mentions Python instead of R
The learning objectives in the introduction mention Python. This course is about R. This should be updated.

### PR DESCRIPTION
**Pull request by mkuzak**
Resolves #3

### CODE REVIEW

File: episodes/introduction.Rmd [hunk 1]
```diff
@@ -5,8 +5,8 @@ exercises: 0
 ---
 
 ::: questions
-- "What is a Python package?"
-- "Why do we want a Python package?"
+- "What is a R package?"
```
Comment (liekelotte) [inline episodes/introduction.Rmd]:
```suggestion
- "What is an R package?"
```

I corrected grammar in this line

File: episodes/introduction.Rmd [hunk 2]
```diff
@@ -5,8 +5,8 @@ exercises: 0
...
+- "Why do we want a R package?"
```
Comment (liekelotte) [inline episodes/introduction.Rmd]:
```suggestion
- "Why do we want an R package?"
```
Corrected grammar as below!

**Review by liekelotte — CHANGES_REQUESTED** [review]
Thank you for changing Python to R, could you make sure the grammar is correct after this review?

**Comment by mkuzak** [comment]
Well spotted. Thanks.  I'll merge your changes.

**Review by liekelotte — APPROVED** [review]
Thanks so much for your great work!

**Pull request merged**